core/tests/protractor_desktop/collections.js

```diff
@@ -36,11 +37,16 @@ describe('Collections', function() {
   var secondExplorationId = null;
   var thirdExplorationId = null;
   var fourthExplorationId = null;
+  var libraryPage = null;
+
+  var saveInProgressLabel = element(by.css(
```

Comment:
I would suggest defining this in CollectionEditorPage.js

Reply:
Moved. Done.